lib/node.js

```diff
@@ -18,10 +18,13 @@ type buffer$NonBufferEncoding =
   'utf16le' | 'UTF16LE' | 'utf-16le' | 'UTF-16LE';
 type buffer$Encoding = buffer$NonBufferEncoding | 'buffer'
 
+type buffer$Source = Array<number> | string | ArrayBuffer | Uint8Array;
```

Comment:
both places this is used also do `| Buffer`. does it make sense to add `Buffer` to this union?